turbo.json

```diff
@@ -12,38 +12,41 @@
     },
     "@realm/bindgen#build": {
       "inputs": [
-        "src/*"
+        "src/",
+        "generated/"
       ],
       "dependsOn": [
         "generate:spec-schema"
       ]
     },
     "@realm/binding#compile": {
       "inputs": [
-        "generated/node/*"
+        "../bindgen/src/"
```

Comment:
We also need to list the `vendor/realm-core` directory here. Otherwise updates to core may not trigger a rebuild.